cves/CVE-2010-4476.yml

```diff
@@ -64,8 +73,8 @@ fixes:
    - commit:
      note:
 vccs:
-  - commit:
-    note:
+  - commit: 407841c426dc52a4c6b8ccd297df6c484a540056
+    note: SVN rev 302975, from the Tomcat website.
```

Comment:
More insight into what was changed here could aid in painting a complete picture for a reader.